libkbfs/bsplitter_simple.go

```diff
@@ -112,6 +114,52 @@ func NewBlockSplitterSimple(desiredBlockSize int64,
 	}, nil
 }
 
+// SetMaxDirEntriesByBlockSize sets the maximum number of directory
+// entries per directory block, based on the maximum block size.  If
+// the `KEYBASE_BSPLIT_MAX_DIR_ENTRIES` is set, this function does
+// nothing.
+func (b *BlockSplitterSimple) SetMaxDirEntriesByBlockSize(
+	codec kbfscodec.Codec) error {
+	dirEnv := os.Getenv("KEYBASE_BSPLIT_MAX_DIR_ENTRIES")
```

Comment:
Hmm would it be better to just call `getMaxDirEntriesPerBlock()` here so we don't have to worry about env var in multiple functions. Although if you agree, we can also change this when we do KBFS-3306.

Reply:
Also I guess I'm not sure how we use this env var. Do we rely on it in tests?

Reply:
> Hmm would it be better to just call getMaxDirEntriesPerBlock() here so we don't have to worry about env var in multiple functions. Although if you agree, we can also change this when we do KBFS-3306.

No, because there's a difference between an unset envvar and the envvar set to "0", and the return value of that function doesn't show the difference.

> Also I guess I'm not sure how we use this env var. Do we rely on it in tests?

No, I just found it useful for local testing, to run all the tests in dir-splitting mode without having to recompile.